docs/shared/concepts/more-concepts.md

```diff
@@ -14,4 +14,4 @@
 - [Grouping Libraries](/more-concepts/grouping-libraries)
 - [Buildable and Publishable Libraries](/more-concepts/buildable-and-publishable-libraries)
 - [How the Project Graph is Built](/more-concepts/how-project-graph-is-built)
-- [Encapsulated Nx and the Nx Wrapper](/more-concepts/encapsulated-nx-and-the-wrapper)
```

Comment:
Do we need a redirect rule for this?

Reply:
No, page hasn't been linked to enough for that to be necessary

Reply:
No published content, etc

Reply:
This one we're fine. It's pretty new and we haven't really talked about it yet.